File: core/Core/Models/Account.cs
using System;
using System.Collections.Generic;

namespace Core.Models;

public partial class Account
{
    public int AccountId { get; set; }

    public int? CustomerId { get; set; }

    public int? TypeId { get; set; }

    public string Iban { get; set; } = null!;

    public decimal? Balance { get; set; }

    public DateTime? CreatedAt { get; set; }

    public bool? IsActive { get; set; }

    public virtual ICollection<Card> Cards { get; set; } = new List<Card>();

    public virtual Customer? Customer { get; set; }

    public virtual ICollection<Transaction> TransactionSourceAccountNavigations { get; set; } = new List<Transaction>();

    public virtual ICollection<Transaction> TransactionTargetAccountNavigations { get; set; } = new List<Transaction>();

    public virtual AccountType? Type { get; set; }
}

File: analytic/domain/account/account.py
from datetime import datetime


class Account:
    def __init__(self, account_id: int, customer_id: int, type_id: int, iban: str, balance: str, created_at: datetime, is_active: bool):
        self.account_id: int = account_id
        self.customer_id: int = customer_id
        self.type_id: int = type_id
        self.iban: str = iban
        self.balance: str = balance
        self.created_at: datetime = created_at
        self.is_active: bool = is_active
